schema/crdb/tuf-trust-root/up01.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS omicron.public.tuf_trust_root (
+    -- Identity metadata (resource)
+    id UUID PRIMARY KEY,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_deleted TIMESTAMPTZ,
+    root_role JSONB NOT NULL
```

Comment:
It seems important that this field reliably round-trip with no relevant changes. Do we know for sure (enough) that storing it as `JSONB` won't ever change it in any way that's meaningful to TUF?